### DISCUSSION TITLE
Triple Colon

### DISCUSSION THREAD

**Discussion by paulushub** [dotnet/docfx#8095] [Q&A]
Hello,
I am using the Markdown custom container or the triple colon and it is not working.

I assume the feature is similar to that described here https://github.com/xoofx/markdig/blob/master/src/Markdig.Tests/Specs/CustomContainerSpecs.md

So, I tried using it as
```
:::media-center
![resource_settings](/images/resource_settings.png)
:::
.
```
However, the triple colon is simply copied to the HTML output. 
What am I doing wrong or this feature is not the same as the custom container?

**Comment by markjulmar** [comment]
Hi @paulushub -- did you find a solution to this? I didn't look too hard since this was posted a while back, but I can say that DocFX doesn't load all the available extensions when it's building up the Markdown processor. This one might conflict with the triplecolon extension model provided by DocFX, but you could try adding the extension as part of building up the pipeline -- [this is the one you'd need](https://github.com/xoofx/markdig/blob/master/src/Markdig/Extensions/CustomContainers/CustomContainerExtension.cs).

**Reply by paulushub** [reply]
@markjulmar Thanks for the reply.
> did you find a solution to this? 

No, the documentation is already up, but I will look into your suggestions next time the docs is up for update.
